book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self
+  end
```

Comment:
Good effort, @Estete9! 👍 Your implementation of the `add_rental` method demonstrates an understanding of the many-to-many relationship. However, there's a small adjustment needed to fully align with the project requirements.

#### Suggested Improvement:

🔍 **Required Changes**
   - The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.`

#### Updated Method:

```ruby
  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```

#### Explanation:
   - This adjustment ensures that the `add_rental` method conforms to the specified project requirements.
   - By including the `person` parameter in the method, you establish the 'has-many' association with the correct parameters.

Keep up the good work! If you have any questions or need further clarification, feel free to ask. 🚀

Best,
Temitope